src/js/msp/MSPHelper.js

```diff
@@ -764,25 +764,32 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 RX_CONFIG.spektrum_sat_bind = data.readU8();
                 RX_CONFIG.rx_min_usec = data.readU16();
                 RX_CONFIG.rx_max_usec = data.readU16();
+                RX_CONFIG.rcInterpolation = 0;
+                RX_CONFIG.rcInterpolationInterval = 0;
+                RX_CONFIG.airModeActivateThreshold = 0;
                 if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
                     RX_CONFIG.rcInterpolation = data.readU8();
                     RX_CONFIG.rcInterpolationInterval = data.readU8();
                     RX_CONFIG.airModeActivateThreshold = data.readU16();
+                    RX_CONFIG.rxSpiProtocol = 0;
+                    RX_CONFIG.rxSpiId = 0;
+                    RX_CONFIG.rxSpiRfChannelCount = 0;
+                    RX_CONFIG.fpvCamAngleDegrees = 0;
                     if (semver.gte(CONFIG.apiVersion, "1.31.0")) {
                         RX_CONFIG.rxSpiProtocol = data.readU8();
                         RX_CONFIG.rxSpiId = data.readU32();
                         RX_CONFIG.rxSpiRfChannelCount = data.readU8();
                         RX_CONFIG.fpvCamAngleDegrees = data.readU8();
-                    } else {
-                        RX_CONFIG.rxSpiProtocol = 0;
-                        RX_CONFIG.rxSpiId = 0;
-                        RX_CONFIG.rxSpiRfChannelCount = 0;
-                        RX_CONFIG.fpvCamAngleDegrees = 0;
                     }
-                } else {
-                    RX_CONFIG.rcInterpolation = 0;
-                    RX_CONFIG.rcInterpolationInterval = 0;
-                    RX_CONFIG.airModeActivateThreshold = 0;
+                }
+
+                if (semver.gte(CONFIG.apiVersion, "1.40.0")) {
```

Comment:
I prefer to include this if inside the if of the 1.31 version. Is more optimum. Maybe in a future too much if one inside other is not the best code. But here is only my opinion.